gcp-deployer/machine_setup_configs.yaml

```diff
@@ -179,10 +23,23 @@ items:
       apt-get install -y \
           socat \
           ebtables \
-          docker.io \
           apt-transport-https \
           cloud-utils \
           prips
+
+      function install_configure_docker () {
+          # prevent docker from auto-starting
+          echo "exit 101" > /usr/sbin/policy-rc.d
+          chmod +x /usr/sbin/policy-rc.d
+          trap "rm /usr/sbin/policy-rc.d" RETURN
+          apt install -y docker.io
```

Comment:
Unless there's an alias I didn't notice, this looks broken. Shouldn't the command be `apt-get`, not `apt`?